README.md

```diff
@@ -36,6 +36,12 @@ If you're building for a non-default architecture, you may need to enable
 the `experimental: "true"` flag to enable `docker pull --platform`: see
 https://docs.docker.com/engine/reference/commandline/pull/#options
 
+By default, the build system assumes the host system runs on x86-64 GNU/Linux and it will use QEMU to emulate non-x86 platforms within Docker. If you are building on a non-x86 host, you can use the `--no-qemu` to disable QEMU entirely or `--use-qemu <PLATFORM>` to force the use of a specific build of QEMU for your platform. The platform argument follows the format `os/arch(/variant)`. For example:
+
+    $ ./build --use-qemu linux/arm64/v8 package-docker buster-amd64 <PATH-TO-WKHTMLTOPDF>
+
+to build an AMD64 package on a AArch64 host.
```

Comment:
```suggestion
to build a package on an AArch64 host.
```
I don't think `amd64` is correct here?

Reply:
Well, that was intentional. That command should be able to compile a working AMD64 package from an AArch64 host and theoretically it should work as there are AArch64 builds of QEMU on aptman's repository.

Reply:
Does this actually work? Either way, won't it make sense to use the expected use case (building on non-x86 platforms) as the example?

Reply:
You're right. I updated README to provide a typical use case.